mlstream/utils.py

```diff
@@ -162,7 +173,9 @@ def prepare_data(cfg: Dict, basins: List) -> Dict:
                     basins=basins,
                     dates=[cfg["start_date"], cfg["end_date"]],
                     forcing_vars=cfg["forcing_attributes"],
-                    seq_length=cfg["seq_length"])
+                    seq_length=cfg["seq_length"],
+                    forcings_file_format=cfg["forcings_file_format"],
+                    concat_static=cfg["concat_static"])
```

Comment:
As I said above, I don't think we need `concat_static` here

Reply:
If `create_h5_files` will never call `LumpedBasin` with `concat_static` as `False` then I guess this can be removed.